rosco-web/pkg_scripts/postInstall.sh

```diff
@@ -27,13 +30,18 @@ remove_temp_dir() {
 }
 
 install_packer() {
-  PACKER_VERSION="1.8.1"
-  local packer_version=$(/usr/bin/packer --version)
+  PACKER_VERSION="1.10.1"
+  local packer_version="$(/usr/bin/packer --version)"
   local packer_status=$?
   if [ $packer_status -ne 0 ] || [ "$packer_version" != "$PACKER_VERSION" ]; then
     wget https://releases.hashicorp.com/packer/${PACKER_VERSION}/packer_${PACKER_VERSION}_linux_amd64.zip
```

Comment:
Huh just noticed this isn't including the target arch... that'd probably be ideal on the amd64 thing here...

Reply:
I don't think we've ever supported making deb packages for ARM, so I didn't think it was necessary. But I can add it if you think it should be here.

Reply:
I've added support for arm64. Not sure if the rest of the package is architecture agnostic though.

Reply:
yeah it's minor - MOST Of the docker builds TODAY should be multi-arch.  I'm seeing increased "Build for ARM!" stuff lately.